source/includes/usage-examples/run-command.c

```diff
@@ -0,0 +1,144 @@
+        mongoc_client_new ("<connection string URI>");
+    collection = mongoc_client_get_collection (client, "<database name>", "collection name");
+
+    {
+        // start-execute-command
+        bson_t reply;
+        bson_error_t error;
+    
+        bson_t *command = BCON_NEW ("hello", BCON_INT32 (1));
+        bool retval = mongoc_client_command_simple (client, "admin", command, NULL, &reply, &error);
+    
+        if (!retval) {
+            fprintf (stderr, "Failed to run command: %s\n", error.message);
+        } else {
+            char *str = bson_as_canonical_extended_json (&reply, NULL);
+            printf ("%s\n", str);
+            bson_free (str);
+        }
+    
+        bson_destroy (command);
+        bson_destroy (&reply);
+        // end-execute-command
+    }
+
+    {
+        // start-execute-command-with-options
+        bson_t reply;
+        bson_error_t error;
+
+        bson_t *command = BCON_NEW(
+            "grantRolesToUser", BCON_UTF8("user011"),
+            "roles", "[", BCON_UTF8("readWrite"), "]"
+        );
+
+        mongoc_write_concern_t *write_concern = mongoc_write_concern_new();
+        mongoc_write_concern_set_w (write concern, MONGOC_WRITE_CONCERN_W_MAJORITY);
+        bson_t *opts = bson_new();
+        mongoc_write_concern_append(write_concern, opts);
+
+        bool retval = mongoc_client_command_with_opts(client, "admin", command, NULL, opts, &reply, &error);
+
+        if (!retval) {
+            fprintf(stderr, "Failed to run command: %s\n", error.message);
+        } else {
+            char *str = bson_as_canonical_extended_json(&reply, NULL);
+            printf("Command Result: %s\n", str);
+            bson_free(str);
+        }
+
+        bson_destroy(command);
+        bson_destroy(opts);
+        bson_destroy(&reply);
+        // end-execute-command-with-options
+    }
+
+    {
+        // start-run-command-example
+        bson_t reply;
+        bson_error_t error;
+
+        bson_t *cmd = BCON_NEW ("cloneCollectionAsCapped",
+                BCON_UTF8 ("my_collection"),
+                "toCollection",
+                BCON_UTF8 ("my_capped_collection"),
+                "size",
+                BCON_INT64 (1024 * 1024));
+
+        /* Includes write concern "majority" in command options */
+        mongo_write_concern_t *write_concern = mongoc_write_concern_new ();
+        mongoc_write_concern_set_w (write_concern, MONGOC_WRITE_CONCERN_W_MAJORITY);
+        bson_t *opts = bson_new ();
+        mongoc_write_concern_append (write_concern, opts);
+
+        if (mongoc_client_write_command_with_opts (client, "test", cmd, opts, &reply, &error)) {
+            char *str = bson_as_canonical_extended_json (&reply, NULL);
+            printf ("cloneCollectionAsCapped: %s\n", str);
+            bson_free (str);
+        } else {
+            fprintf (stderr, "cloneCollectionAsCapped: %s\n", error.message);
+        }
+
+        bson_free (cmd);
+        bson_free (opts);
```

Comment:
```suggestion
        bson_free (opts);
        bson_destroy (&reply);
```

Destroy `reply` before reusing to avoid memory leak.